### PR TITLE
make some string in machine and dashboard to be translated

### PR DESCRIPTION
**Pull request by vanloswang**

### CODE REVIEW

File: pkg/sosreport/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     "tools": {
         "index": {
-            "label": "Diagnostic report"
+            "label": "Diagnostic reports"
```
Comment (petervo) [inline pkg/sosreport/manifest.json]: @andreasn are you ok with the pluralization change here?

Comment (vanloswang) [inline pkg/sosreport/manifest.json]: @petervo The UI looks OK for me.  As I know, The manifest.json is just for UI layout.

Comment (petervo) [inline pkg/sosreport/manifest.json]: Yes, makes sense to match the page title.

Comment (andreasn) [inline pkg/sosreport/manifest.json]: Yeah, sounds good to me!

**Pull request closed without merging**